tools/rosgraph/CMakeLists.txt

```diff
@@ -17,4 +17,5 @@ catkin_install_python(PROGRAMS scripts/rosgraph
 
 if(CATKIN_ENABLE_TESTING)
   catkin_add_nosetests(test)
+  catkin_add_nosetests(test/isolated/test_roslogging_user_logger.py)
```

Comment:
This line shouldn't be necessary. The line above already collects all tests in the `test` folder.